.k8s_ci.Dockerfile

```diff
@@ -41,21 +41,15 @@ ENV LANG=en_US.UTF-8
 
 # Install build dependencies + ansible
 RUN set -xe; \
-    echo "force-unsafe-io" > /etc/dpkg/dpkg.cfg.d/02apt-speedup \
-    && echo "Acquire::http {No-Cache=True;};" > /etc/apt/apt.conf.d/no-cache \
+    echo "Acquire::http {No-Cache=True;};" > /etc/apt/apt.conf.d/no-cache \
     && apt-get -qq update && apt-get install -y --no-install-recommends \
         locales locales-all \
-        apt-transport-https \
         git \
         make \
-        libpython3.6 \
-        python3-dev \
-        python3-virtualenv \
-        software-properties-common \
-        ssh \
+        libc-dev \
+        bzip2 \
         gcc \
-    && apt-get -qq update && apt-get install -y --no-install-recommends \
-        ansible \
+    && pip install --no-cache virtualenv 'ansible>=2.9,<2.10' \
```

Comment:
Does ansible need to be pinned?

Reply:
https://github.com/galaxyproject/ansible-galaxy/blob/main/meta/main.yml
Eh, apparently not the way I did it. Sorry.

Reply:
Done. Pinned to `>=2.7`